### PR TITLE
Some more proper framework crap

### PR DESCRIPTION
**Pull request by Cryowatt**

### CODE REVIEW

File: AdventOfCode.Runner/Input/README.MD [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿Drop your input files here with the name `Day1.txt`, `Day2.txt`, etc.
```
Comment (shawnkoon) [inline AdventOfCode.Runner/Input/README.MD]: Interesting does `.txt` file just need to be some kind of text file with strings?

Comment (Cryowatt) [reply AdventOfCode.Runner/Input/README.MD:1]: AoC usually provides the user-specific input as text. That's why I've added a .gitignore entry for these, so nobody accidentally checks it in.

Comment (Cryowatt) [reply AdventOfCode.Runner/Input/README.MD:1]: If you look at my `Day<>` design you can see I specify a function to convert string into a more-appropriate input format. This keeps the code cleaner when you want to do stupid shit like solve these problems in a single-line linq statement.

Comment (shawnkoon) [reply AdventOfCode.Runner/Input/README.MD:1]: haha i see nice work!

**Pull request merged**